app/views/rb_taskboards/show.html.erb

```diff
@@ -124,6 +134,50 @@ See doc/COPYRIGHT.rdoc for more details.
           <% end %>
         </tr>
       <% end %>
+      <% unless @subprojects.nil? %>
+        <% @subprojects.each do |subproject| %>
+          <% @sprint.stories(subproject).each do |story| %>
+            <tr class="<%= story_html_id_or_empty(story) %>">
+              <td>
+                <div class="story <%= mark_if_closed(story) %>" id="<%= story_html_id_or_empty(story) %>">
+                  <div class='story-bar'>
+                    <div class="status">
+                      <%= story.status.name %>
+                    </div>
+                    <div class="id">
+                      <%= work_package_link_or_empty(story) %>
+                    </div>
+                  </div>
+                  <div class="subject"><%= story.subject %></div>
+                  <div class='story-footer'>
+                    <div class="assigned_to_id">
+                      <% if story.assigned_to.present? %>
+                        <%= link_to_user(story.assigned_to) %>
+                      <% else %>
+                        <em><%= l('backlogs.unassigned') %></em>
+                      <% end %>
+                    </div>
+                    <div class="story-points">
+                      <%= story.story_points %>
+                    </div>
+                  </div>
+                </div>
+              </td>
+              <% if User.current.allowed_to?(:create_tasks, @project) %>
+                <td class ="add_new clickable">+</td>
+              <% else %>
+                <td class ="add_new"></td>
+              <% end %>
+              <% @statuses.each do |status| %>
+                <td class="swimlane list <%= status.is_closed? ? 'closed' : '' %>" id="<%= story.id %>_<%= status.id %>">
+                  <%= render :partial => "rb_tasks/task",
+                             :collection => story.tasks.select { |task| task.status.id == status.id } %>
+                </td>
+              <% end %>
+            </tr>
+          <% end %>
+        <% end %>
+      <% end %>
```

Comment:
I don't think this is necessary if you fetch all the stories in the controller and handle them uniformly here in the view.

If it is necessary, we should try to not have duplicate code but to reuse what we have instead and generalize it.

 The `@project` in https://github.com/finnlabs/openproject-backlogs/pull/233/files#diff-8f682c21049311989a264bc4253eba95R166 is incorrect IMO because it would check the permission in the current project and not in the subproject.